.ci/scripts/test.sh

```diff
@@ -1,18 +1,22 @@
 #!/usr/bin/env bash
-set -xeo pipefail
+set -xo pipefail
 STACK_VERSION=${STACK_VERSION:-8.6.1}
 
 pip install docker-compose>=1.25.4
 
-USER_ID="$(id -u):$(id -g)" \
-NODEJS_VERSION=$(cat .nvmrc) \
-STACK_VERSION=${STACK_VERSION} \
-docker-compose \
-  -f ./dev-utils/docker-compose.yml \
-  --log-level INFO \
-  up \
-  --quiet-pull \
-  --exit-code-from node-puppeteer \
-  --remove-orphans \
-  node-puppeteer
+for i in {1..3};
```

Comment:
Can you explain what's the reason for this with a comment?
OTOH, do you think a trap will be needed to clean up the previous run? Or will `--remove-orphans` is enough?

Reply:
I added a comment to explain the logic.
The `--remove-orphans` flag should be enough to clean up everything before retrying.
We were using the same logic in the Jenkins implementation.